aspoet/src/main/kotlin/com/google/androidstudiopoet/generators/SourceModuleGenerator.kt

```diff
@@ -56,15 +77,15 @@ class SourceModuleGenerator(private val moduleBuildGradleGenerator: ModuleBuildG
             val allJobs = mutableListOf<Job>()
             projectBlueprint.moduleBlueprints.asReversed().forEach { blueprint ->
                 val job = launch {
-                    writeModule(blueprint)
+                    writeModule(blueprint, projectBlueprint.generateBazelFiles)
```

Comment:
Please, make this boolean a part of the `ModuleBlueprint` and `AndroidModuleBlueprint`, then `buildBazelBlueprint` can be optional in both of them and if it is present, then the `BUILD.bazel` file is generated by respective generator